### ISSUE TITLE
During a2c training, why use advantage as the target for actor?

### ISSUE THREAD

**Issue by lx231818826** [flyyufelix/VizDoom-Keras-RL#4]
Hi
Nice job. When I tried to understand the a2c.py, I noticed that you are using advantage as the target to train the actor:

        actor_loss = self.actor.fit(update_inputs, advantages, nb_epoch=1, verbose=0)

But as I understood, shouldn't it use the policy that derives from softmax(value_predicted) to train?

Thanks!!

**Comment by lx231818826** [comment]
Oh I see the categorical_crossentropy in the loss function. Never mind, I'll close the post.
